lib/outbound/test.js

```diff
@@ -10,7 +10,7 @@ describe('Outbound', function(){
   var outbound;
   var analytics;
   var options = {
-    publicApiKey: 'pub-3e2b0899b2c81c6f0c59342d1ff057c3'
+    publicApiKey: 'pub-9cb1d6e54b003d5274e54a483ef741be'
```

Comment:
We should leave this as the old key right @amillet89? I'm guessing this one is @usmanity's personal Outbound.io key?